### DISCUSSION TITLE
Using Fader for lights

### DISCUSSION THREAD

**Discussion by jlr91** [maartentamboer/X-Touch-Mini-FS2020#96] [Q&A]
Hi, I'd like to use the fader for lights.
The command with Mobiflight WASM module is : @ 1.2700 / 0 max 100 min 3 (>K:2:LIGHT_POTENTIOMETER) where  1.2700 / 0 max 100 min is the first parameter in order to transform the light potentiometer scale 0-100 into the fader scale from 0 to 127 and 3 being the second parameter (the panel light potentiometer for C172).
It doesn't work  because of the min and max value needed in the fader configuration, I think.
How can I use it ?

**Comment by JLD95** [comment]
Hi,
I search exactly the same function:  In my config_a320;json  I use 

  "faders": [
    {
      "index": 1,
      "event_change": "LIGHT POTENTIOMETER:7",
      "min_value": 0,
      "max_value": 100
    }
  ], 

but at run I have WARNING: Event LIGHT POTENTIOMETER:7, was not found in simconnect list. Using a manual binding
How to make this binding ?
